src/server/app.ts

```diff
@@ -48,6 +51,27 @@ function addRoutes(attach: string, router: Router | Handler): void {
   app.use(SERVER_SETTINGS.getServerRoot() + attach, router);
 }
 
+// Add request metrics
+app.use((req: Request, res: Response, done) => {
+  const stopRequestTimer = metrics.requestDuration.startTimer();
+
+  function addRequestMetrics() {
+    const lables = { host: req.hostname, route: req.path, method: req.method, status: res.statusCode };
```

Comment:
The `host` label won't be very useful as the proxy does not rewrite the `Host` header. We can simply remove this as Prometheus adds the `instance` label automatically. My fault for not thinking of this before, sorry.